<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserSwaps</class>
 <widget class="QMainWindow" name="UserSwaps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Обмены</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbMySwaps">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Предложил я</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbOthersSwaps">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Предложили мне</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frameMy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <widget class="QPushButton" name="btnDelete">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="twMySwaps">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>641</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="twOthersSwaps">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>641</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frameOthers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>101</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <widget class="QPushButton" name="btnSwap">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Обменять</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOthersBook">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>К книге</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnMenu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Меню</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
